acc_to_csv.cpp

```diff
@@ -63,16 +64,23 @@ int main(int argc, char** argv) {
 
     //auto location = wrapper.getDataForCurrentEntry(PTF::Gantry1);                                                                                                                                                                           
 
-    //csv << location.x << "," << location.y << "," << location.z << ",";                                                                                                                                                                     
+    //csv << location.x << "," << location.y << "," << location.z << ",";                                                                                                                         for (unsigned int i = 0; i < wrapper.getNumEntries(); i++) {
+     auto time_before=wrapper.getReadingTime();
+     wrapper.setCurrentEntry(0);
+     auto time_after=wrapper.getReadingTime();
+
+     csv <<time_before.time_c-time_after.time_c << "," ;
+      
 
     for (int phidget : phidgets) {
 
 	       auto reading  = wrapper.getReadingForPhidget(phidget);
 
 	       csv << reading.Ax[0] << "," << reading.Ay[0] << "," << reading.Az[0];
 
-	       if (phidget != 100) {
-	 	csv << ",";
+	       if (phidget == 100) {
+			   auto acceleration=wrapper.getReadingAcceleration();
+			    csv << acceleration.acc_x[0] << "," << reading.acc_y[0] << "," << reading.acc_z[0];
```

Comment:
This line doesn't compile for me.  Better would be 

               if (phidget == 100) {
                           auto acceleration=wrapper.getReadingAcceleration();
-                           csv << acceleration.acc_x[0] << "," << reading.acc_y[0] << "," << reading.acc_z[0];
+                           csv << acceleration.acc_x << "," << acceleration.acc_y << "," << acceleration.acc_z;
               }
             }